pkgs/applications/window-managers/gamescope/default.nix

```diff
@@ -73,8 +86,20 @@ stdenv.mkDerivation {
     pipewire
     libcap
     stb
+    hwdata
+    openvr
+    vkroots
+    libdisplay-info
   ];
 
+  postUnpack = ''
+    rm -rf source/subprojects/vkroots
+    mkdir source/subprojects/vkroots
+    ln -s ${vkroots}/* source/subprojects/vkroots
+    chmod -R +r+w source/subprojects/vkroots
+    chown -R $USER: source/subprojects/vkroots
```

Comment:
```suggestion
    ln -s ${vkroots}/* source/subprojects/vkroots
    chmod -R +r+w source/subprojects/vkroots
    chown -R $USER: source/subprojects/vkroots
```
Why not copy?

Reply:
I just linked the whole directory now. Linking is way faster than copying